bears/general/CPDBear.py

```diff
@@ -91,7 +89,8 @@ def run(self, language: str, minimum_tokens: int=20,
         files = ','.join(self.file_dict.keys())
         executable = which('pmd') or which('run.sh')
         arguments = ('bash', executable, 'cpd', '--skip-lexical-errors',
-                     '--minimum-tokens', str(minimum_tokens),
+                     '--minimum-tokens', str(
+                            Redundancy.Clone.min_clone_tokens.default),
```

Comment:
Maybe we should write a decorator for that: something like 
````
@linter(...)
@set_aspects( {'Redundancy.Clone.minimun_tokens':'1}
```
Having something like 
```
LANGUAGES = {....}
ASPECTS = { #a set of aspectbase element} 
```